src/test/API.test.js

```diff
@@ -26,7 +26,7 @@ describe('User API methods', () => {
   // Make sure we clean up.
   afterAll(async (done) => {
     // eslint-disable-next-line no-underscore-dangle
```

Comment:
This should also be deleted. Also, I think we should make comments for eslint with `/* */` and other comments with `//` and not mix them